src/metabase/cache/api.clj

```diff
@@ -196,7 +196,7 @@
                                        :with-overrides? (= include :overrides)})]
     {:status (if (= cnt -1) 404 200)
      :body   {:count   cnt
-              :message (case [(= include :overrides) (if (pos? cnt) 1 cnt)]
+              :message (condp = [(= include :overrides) (if (pos? cnt) 1 cnt)]
```

Comment:
This produces hash collisions with `case` which I'm not sure how to avoid otherwise.